### DISCUSSION TITLE
Milvus faling to connect to etcd

### DISCUSSION THREAD

**Discussion by rmaknikar** [milvus-io/milvus#32736] [Q&A and General discussion]
Deployed Milvus on EKS cluster. The Session try to connect to etcd but failing

[DEBUG] [sessionutil/session_util.go:257] ["Session try to connect to etcd"]
{"level":"warn","ts":"2024-04-30T06:56:57.203Z","logger":"etcd-client","caller":"v3@v3.5.5/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc0001e6a80/milvus-dev-milvus-dev-etcd:2379","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing: dial tcp: lookup milvus-dev-milvus-dev-etcd on 172.20.0.10:53: no such host\""}

**Comment by xiaofan-luan** [comment]
milvus-dev-milvus-dev-etcd on 172.20.0.10:53: no such host

seems that milvus can not connect to your etcd due to network issue

**Comment by LoveEachDay** [comment]
@rmaknikar Could you double check that etcd service are running well?  If you are using a kubernetes service name, please make sure the service object has been successfully created. 
Please following the instructions from https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution to debugging the dns resolution.
